README.md

```diff
@@ -54,7 +54,7 @@ Note: These OVAs are not updated for security fixes and it is recommended to alw
 
 | Kubernetes | CentOS 7 | Ubuntu 18.04 | Ubuntu 20.04 | Photon 3 |
 | :--------: | :------: | :----------: | :----------: | :------: |
-|  v1.21.11   |   [ova](https://storage.googleapis.com/capv-images/release/v1.21.11/centos-7-kube-v1.21.11.ova), [sha256](https://storage.googleapis.com/capv-images/release/v1.21.11/centos-7-kube-v1.21.11.ova.sha256)   |   [ova](https://storage.googleapis.com/capv-images/release/v1.21.11/ubuntu-1804-kube-v1.21.11.ova), [sha256](https://storage.googleapis.com/capv-images/release/v1.21.11/ubuntu-1804-kube-v1.21.11.ova.sha256)   |   [ova](https://storage.googleapis.com/capv-images/release/v1.21.11/ubuntu-2004-kube-v1.21.11.ova), [sha256](https://storage.googleapis.com/capv-images/release/v1.21.11/ubuntu-2004-kube-v1.21.11.ova.sha256)| [ova](https://storage.googleapis.com/capv-images/release/v1.21.11/photon-3-kube-v1.21.11.ova), [sha256](https://storage.googleapis.com/capv-images/release/v1.21.11/photon-3-1804-kube-v1.21.11.ova.sha256) |
+|  v1.21.11   |   [ova](https://storage.googleapis.com/capv-images/release/v1.21.11/centos-7-kube-v1.21.11.ova), [sha256](https://storage.googleapis.com/capv-images/release/v1.21.11/centos-7-kube-v1.21.11.ova.sha256)   |   [ova](https://storage.googleapis.com/capv-images/release/v1.21.11/ubuntu-1804-kube-v1.21.11.ova), [sha256](https://storage.googleapis.com/capv-images/release/v1.21.11/ubuntu-1804-kube-v1.21.11.ova.sha256)   |   [ova](https://storage.googleapis.com/capv-images/release/v1.21.11/ubuntu-2004-kube-v1.21.11.ova), [sha256](https://storage.googleapis.com/capv-images/release/v1.21.11/ubuntu-2004-kube-v1.21.11.ova.sha256)| [ova](https://storage.googleapis.com/capv-images/release/v1.21.11/photon-3-kube-v1.21.11.ova), [sha256](https://storage.googleapis.com/capv-images/release/v1.21.11/ubuntu-1804-kube-v1.21.11.ova.sha256) |
```

Comment:
Are we now linking the ubuntu sha for the photon image?

Reply:
uh oh totally missed it the wrong way around :D :thx